### ISSUE TITLE
2.9.0 build with libcloudproviders is broken

### ISSUE THREAD

**Issue by antonio-rojas** [owncloud/client#8996]
### Expected behaviour
It builds

### Actual behaviour
```
/build/owncloud-client/src/client/src/gui/libcloudproviders/libcloudproviders.cpp:85:52: error: ‘class OCC::FolderMan’ has no member named ‘list’
   85 |     const auto newFolders = FolderMan::instance()->list();
      |                                                    ^~~~
```

`FolderMan::list()` was removed in commit 3f5163a4f30cf5978ffd7ee363e0d81065eb29c0

**Comment by TheOneRing** [comment]
I'd recommend to disable the feature for now as it not really provides any benefits.
But we will fix it with 2.9.1

**Comment by TheOneRing** [comment]
What is the current state of libcloudproviders, does it by now provide value.
The only available documentation is what its supposed to provide one day form 2015 
https://csorianognome.wordpress.com/2015/07/07/cloud-providers/ .
https://gitlab.gnome.org/World/libcloudproviders

**Comment by TheOneRing** [comment]
Fixed in 2.9.2
